examples/ecs/Scarb.toml

```diff
@@ -18,5 +18,5 @@ initializer_class_hash = "0xbeef"
 rpc_url = "http://localhost:5050/"
 
 # Default account for katana with seed = 0
-account_address = "0x03ee9e18edc71a6df30ac3aca2e0b02a198fbce19b7480a63a0d71cbd76652e0"
-private_key = "0x0300001800000000300000180000000000030000000000003006001800006600"
+account_address = "0x06f62894bfd81d2e396ce266b2ad0f21e0668d604e5bb1077337b6d570a54aea"
+private_key = "0x078e6e3e4a50285be0f6e8d0b8a61044033e24023df6eb95979ae4073f159ae6"
```

Comment:
> By the way, i was trying to migrate the world to katana starknet local node, but i receive this beautiful error:
> 
> ```
>  WARN  katana_rpc::starknet::rpc] Estimate fee error: TransactionExecution(ValidateTransactionError(VirtualMachineExecutionErrorWithTrace { trace: "Error in the called contract (0x06f62894bfd81d2e396ce266b2ad0f21e0668d604e5bb1077337b6d570a54aea):\nError at pc=0:124:\nSignature 019ea49664ed9c83988a68c18254356b92e67e9ce9ddf747e02aa888d1cf0aa304889ddbf99fd2377b4e62175cad39123b31a43cc571c54c89de2cbd1559c95a, is invalid, with respect to the public key 3417843094722320147644179919180496171252644426037268865213458836300447980262, \n    and the message hash 1534993943596222229563830380315979390987450542602141400586751790671276568433.\nCairo traceback (most recent call last):\nUnknown location (pc=0:615)\nUnknown location (pc=0:600)\nUnknown location (pc=0:245)\n", source: CairoRunError(VmException(VmException { pc: 124, inst_location: None, inner_exc: Memory(InvalidSignature(("019ea49664ed9c83988a68c18254356b92e67e9ce9ddf747e02aa888d1cf0aa304889ddbf99fd2377b4e62175cad39123b31a43cc571c54c89de2cbd1559c95a", 3417843094722320147644179919180496171252644426037268865213458836300447980262, 1534993943596222229563830380315979390987450542602141400586751790671276568433))), error_attr_value: None, traceback: Some("Cairo traceback (most recent call last):\nUnknown location (pc=0:615)\nUnknown location (pc=0:600)\nUnknown location (pc=0:245)\n") })) }))
> ```
> 
> Any hints?

Seems like you have copied the public key instead of the private key.

```console
PREFUNDED ACCOUNTS
==================

| Account address |  0x06f62894bfd81d2e396ce266b2ad0f21e0668d604e5bb1077337b6d570a54aea 
| Private key     |  0x07230b49615d175307d580c33d6fda61fc7b9aec91df0f5c1a5ebe3b8cbfee02
| Public key      |  0x078e6e3e4a50285be0f6e8d0b8a61044033e24023df6eb95979ae4073f159ae6
```

Reply:
Oh, that was such a bad mistake, you are right, this was the reason why it didn't work.